_projects/_template.md

```diff
@@ -5,7 +5,7 @@ published: false # change to true once ready to post
 
 title: "" # project title inside quotes
 excerpt: "" # shows on project list page
-seo_description: "" # goes in project meta description
+seo_description: "Azavea Open Source software engineering fellows choose from impactful open source projects as part of the application process." # goes in project meta description
```

Comment:
If we are trying to make these all the same, I think the better approach may be to eliminate `seo_description` from the templates (and the existing projects) and update `default.html` so that it uses the top-level `description` variable.

Reply:
Something like:

```diff
diff --git a/_layouts/default.html b/_layouts/default.html
index 7cf5af8..84db174 100644
--- a/_layouts/default.html
+++ b/_layouts/default.html
@@ -8,7 +8,7 @@
   
   <!-- SEO -->
   <title>{{ site.title }}{% if page.title %} - {{ page.title }}{% endif %}</title>
-  <meta name="description" content="{% if page.seo_description %}{{ page.seo_description }}{% endif %}">
+  <meta name="description" content="{{ site.description }}">
 
   <link rel="apple-touch-icon" sizes="180x180" href="{{ site.baseurl }}/assets/favicon/apple-touch-icon.png">
   <link rel="icon" type="image/png" sizes="32x32" href="{{ site.baseurl }}/assets/favicon/favicon-32x32.png">
diff --git a/_projects/_template.md b/_projects/_template.md
index 9ef8b76..e85f710 100644
--- a/_projects/_template.md
+++ b/_projects/_template.md
@@ -5,7 +5,6 @@ published: false # change to true once ready to post
 
 title: "" # project title inside quotes
 excerpt: "" # shows on project list page
-seo_description: "Azavea Open Source software engineering fellows choose from impactful open source projects as part of the application process." # goes in project meta description
 
 # project attributes
 requirements: # bullet list of requirements – one requirement per line – follow below format
diff --git a/_projects/ashlar.md b/_projects/ashlar.md
index bc701e9..43caef3 100644
--- a/_projects/ashlar.md
+++ b/_projects/ashlar.md
@@ -3,7 +3,6 @@ published: false # change to true once ready to post
 
 title: "Ashlar" # project title inside quotes
 excerpt: "Flexible schema framework for geospatial data" # shows on project list page
-seo_description: "Azavea Open Source software engineering fellows choose from impactful open source projects as part of the application process." # goes in project meta description
 
 # project attributes
 requirements: # bullet list of requirements – one requirement per line – follow below format
diff --git a/_projects/cloud_detection.md b/_projects/cloud_detection.md
index 2ee4d66..a7d99dd 100644
--- a/_projects/cloud_detection.md
+++ b/_projects/cloud_detection.md
@@ -3,7 +3,6 @@ published: true # change to true once ready to post
 
 title: "Cloud detection with neural networks" # project title inside quotes
 excerpt: "Apply neural network techniques being researched by Azavea staff to the problem of cloud detection in satellite imagery." # shows on project list page
-seo_description: "Azavea Open Source software engineering fellows choose from impactful open source projects as part of the application process." # goes in project meta description
 
 # project attributes
 requirements: # bullet list of requirements – one requirement per line – follow below format
diff --git a/_projects/lambda_tiles.md b/_projects/lambda_tiles.md
index a551f1f..66d1f3f 100644
--- a/_projects/lambda_tiles.md
+++ b/_projects/lambda_tiles.md
@@ -3,7 +3,6 @@ published: false
 
 title: "Lambda Tilegarden" # project title inside quotes
 excerpt: "Lambda-based raster and vector tile generation from PostGIS data" # shows on project list page
-seo_description: "Azavea Open Source software engineering fellows choose from impactful open source projects as part of the application process." # goes in project meta description
 
 # project attributes
 requirements: # bullet list of requirements – one requirement per line – follow below format
diff --git a/_projects/roman_tobler.md b/_projects/roman_tobler.md
index 52a31d0..628e59f 100644
--- a/_projects/roman_tobler.md
+++ b/_projects/roman_tobler.md
@@ -3,7 +3,6 @@ published: true # change to true once ready to post
 
 title: "Foot Steps of Rome: Calculating the Cost of Travel in the Roman Empire" # project title inside quotes
 excerpt: "Calculate the cost of traveling between ancient Roman population centers" # shows on project list page
-seo_description: "Azavea Open Source software engineering fellows choose from impactful open source projects as part of the application process." # goes in project meta description
 
 # project attributes
 requirements: # bullet list of requirements – one requirement per line – follow below format
diff --git a/_projects/topojson_vector_data.md b/_projects/topojson_vector_data.md
index 565a4b2..0cfc01d 100644
--- a/_projects/topojson_vector_data.md
+++ b/_projects/topojson_vector_data.md
@@ -3,7 +3,6 @@ published: true # change to true once ready to post
 
 title: "Implement topojson reader/writer for vector data"
 excerpt: "A Scala implementation of the TopoJSON format in GeoTrellis"
-seo_description: "Azavea Open Source software engineering fellows choose from impactful open source projects as part of the application process."
 page-class: projects
 
 # project attributes
diff --git a/page-about.md b/page-about.md
index fd23d5f..a9368cd 100644
--- a/page-about.md
+++ b/page-about.md
@@ -2,7 +2,6 @@
 layout: secondary
 permalink: "/about/"
 title: "About"
-seo_description: "The Azavea Open Source Fellowship Program is a 12-week professional training fellowship that matches software engineering fellows with impactful open source projects."
 page_class: about
 use-intro-imagery: true
 
diff --git a/page-faq.md b/page-faq.md
index c42dd8d..51648cd 100644
--- a/page-faq.md
+++ b/page-faq.md
@@ -2,7 +2,6 @@
 layout: secondary
 permalink: "/faq/"
 title: "FAQ"
-seo_description: "The Azavea Open Source Fellowship Program is a 12-week professional training fellowship that matches software engineering fellows with impactful open source projects."
 page_class: ""
 
 
diff --git a/page-home.html b/page-home.html
index 6f9995c..c2c3203 100644
--- a/page-home.html
+++ b/page-home.html
@@ -2,7 +2,6 @@
 layout: default
 permalink: "/"
 title: "Home"
-seo_description: "The Azavea Open Source Fellowship Program is a 12-week professional training fellowship that matches software engineering fellows with impactful open source projects."
 page_class: homepage
 partials_location: "_includes/_home/"
 js_dependencies:
diff --git a/page-projects.html b/page-projects.html
index f27c500..086a052 100644
--- a/page-projects.html
+++ b/page-projects.html
@@ -2,7 +2,6 @@
 layout: secondary
 permalink: "/projects/"
 title: "Projects"
-seo_description: "The Azavea Open Source Fellowship Program is a 12-week professional training fellowship that matches software engineering fellows with impactful open source projects."
 page_class: projects
 partials_location: "_includes/_projects/"
 js_dependencies:
```

Reply:
Thanks @hectcastro. @designmatty is this something you would like to take a look at?

Reply:
I just addressed it in d28d4257662fdd7ae68415c1b157518dbacd745b (if using just one description is the intended result).

Reply:
thanks for addressing that @hectcastro. Your commit looks good to me. We will need to make sure to update the `description` in the `config.yml`

Reply:
Dan's got that covered in the larger PR.